src/app/modules/form/admin-form/__tests__/admin-form.routes.spec.ts

```diff
@@ -4748,8 +4746,8 @@ describe('admin-form.routes', () => {
       const response = await request
         .get(`/${defaultForm._id}/adminform/submissions/download`)
         .query({
-          startDate: format(subDays(now, 4), 'yyyy-MM-dd'),
-          endDate: format(subDays(now, 3), 'yyyy-MM-dd'),
+          startDate: '2020-02-03',
+          endDate: '2020-02-04',
```

Comment:
should we save these as a string to enforce consistency between `submission.created` and the query?

Reply:
good point, will do